index.html

```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
       </section>
       <section class="contact">
         <p class="contact-title desktop">Let's chat!</p>
-        <p class="contact-title mobile">CONTACT US</p>
+        <p class="contact-title mobile">CONTACT ME</p>
         <p class="contact-description">
           If you have an application you are interested in developing, a feature
           that you need built or a project that needs coding. I’d love to help
           with it
         </p>
-        <form class="contact-form">
+        <form
+          class="contact-form"
+          action="https://formspree.io/f/xvolvllr"
+          method="POST"
+        >
           <input
             type="text"
             id="fullname"
             name="fullname"
             placeholder="Enter Full Name"
+            maxlength="30"
           />
           <div id="names">
             <input
               type="text"
               id="fname"
               name="firstname"
               placeholder="Enter First Name"
+              maxlength="30"
+              required
             />
             <input
               type="text"
               id="lname"
               name="lastname"
               placeholder="Last Name"
+              maxlength="30"
+              required
             />
           </div>
           <input
-            type="text"
+            type="email"
             id="email"
             name="Email"
             placeholder="Email Address"
+            required
           />
-          <textarea rows="5" cols="50" name="description">
-          Write your message here
-      </textarea
+          <!-- <textarea
+            placeholder="Write your message here..."
+            rows="5"
+            cols="50"
+            name="description"
+            maxlength="500"
+            required
           >
+          </textarea> -->
+
+          <textarea
+            id="msg"
+            name="user_message"
+            placeholder="Your message"
+            rows="5"
+            cols="50"
+            required
+            minlength="1"
+            maxlength="500"
+          ></textarea>
+
           <button type="submit" class="send-button">Send</button>
         </form>
       </section>
-
+      <div class="divider-full"></div>
       <footer class="second-footer">
         <ul>
           <li>
             <a href="https://github.com/Marlyn_Mayienga"
```

Comment:
- [ ] Please increase the size for the social media icons and make sure it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166552255-497ae616-16b3-4c63-9f18-49404162329f.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166552283-f064a481-4804-49a7-a730-3dd231750e2e.png)